packages/harpguru-core/src/components/quiz-question-display/hooks/use-quiz-question-cycle/use-quiz-question-cycle.ts

```diff
@@ -1,13 +1,12 @@
 import { useGlobal, useDispatch } from 'reactn'
 import { unstable_batchedUpdates } from 'react-native'
 import { useState, useEffect } from 'react'
-import { getHarpStrata, getPropsForHarpStrata } from 'harpstrata'
-import type { HarpStrata } from 'harpstrata'
 import { DegreeIds, isPitchId } from 'harpparts'
 import type { PitchIds } from 'harpparts'
 
 import { getNextQuizQuestion, hasToggledIncorrectCell } from '../../utils'
 import { ExperienceModes } from '../../../../types'
+import { useFlushBufferedActivityTogglesSingleDispatch } from '../../../../hooks'
```

Comment:
We don't need the "SingleDispatch" suffix now. This hook file can just return a single function.

Reply:
Done